hammer.sh

```diff
@@ -544,6 +574,8 @@ function install_deps_AppImageKit()
       curl -OL https://raw.github.com/probonopd/AppImageKit/master/md5.c
       curl -OL https://raw.github.com/probonopd/AppImageKit/master/md5.h
       curl -OL https://raw.github.com/probonopd/AppImageKit/master/runtime.c
+      #AppImageKit isn't smart enough to find debian library locations, let's help it.
+      sed -i 's|"/usr/lib64"|"/usr/lib" "/usr/lib64" "/usr/lib/i386-linux-gnu" "/usr/lib/x86_64-linux-gnu"|' CMakeLists.txt
```

Comment:
Pretty sure. This is just to build a few fragments of code from AppImageKit and it has no other use. This edit allows it to properly do that one thing.